lib/lambda_ethereum_consensus/p2p/block_downloader.ex

```diff
@@ -3,54 +3,25 @@ defmodule LambdaEthereumConsensus.P2P.BlockDownloader do
   This module requests blocks from peers.
```

Comment:
This module was a GenServer, but since I changed it to download blocks only when asked, I removed that. The problem is that now we have to pass the `host` every time. Would it make sense to store the host in a centralized place, maybe PeerBook?

Reply:
nice!